cmd/internal/cli/action_flags.go

```diff
@@ -226,7 +226,7 @@ var actionHostnameFlag = cmdline.Flag{
 var actionNetworkFlag = cmdline.Flag{
 	ID:           "actionNetworkFlag",
 	Value:        &Network,
-	DefaultValue: "bridge",
+	DefaultValue: "",
```

Comment:
This affects the help message.  Add `(default "bridge")` explicitly at the end of the end of `Usage` to make up for it.

Reply:
Adding ``(default "bridge")`` would suggest that specifying ``--network bridge`` and omitting ``--network`` have the same behavior, which after this merge is only true when `--net` is also specified. The default behavior when `--net` is specified is noted on the previous line in the help message:
```
...
  -n, --net                           run container in a new network
                                      namespace (sets up a bridge network
                                      interface by default)
      --network string                specify desired network type
                                      separated by commas, each network
                                      will bring up a dedicated interface
                                      inside container
      --network-args strings          specify network arguments to pass to
                                      CNI plugins
...
```

Reply:
ok